Makefile

```diff
@@ -631,6 +633,79 @@ enwiki_tuning_reports: \
 	tuning_reports/enwiki.damaging.md \
 	tuning_reports/enwiki.goodfaith.md
 
+############################# English Wiktionary ################################
+
+datasets/enwiktionary.sampled_revisions.200k_2016.json:
+	wget -qO- https://quarry.wmflabs.org/run/97131/output/0/json-lines?download=true > $@
+
+datasets/enwiktionary.autolabeled_revisions.200k_2016.json: \
+		datasets/enwiktionary.sampled_revisions.200k_2016.json
+	cat $< | \
+	./utility autolabel --host=https://en.wiktionary.org \
+		--trusted-groups=sysop,oversight,bot,rollbacker,checkuser,abusefilter,bureaucrat \
+		--trusted-edits=1000 \
+		--revert-radius=3 \
+		--revert-window=48 \
+		--verbose > $@
+
+datasets/enwiktionary.revisions_for_review.5k_2016.json: \
+		datasets/enwiktionary.autolabeled_revisions.200k_2016.json
+	( \
+	 cat $< | \
+	 grep '"needs_review": true' | \
+	 shuf -n 2500; \
+	 cat $< | \
+	 grep '"needs_review": false' | \
+	 shuf -n 2500 \
+	) | shuf > $@
```

Comment:
If we're restarting, let's not do this at all.

Reply:
Done!